### ISSUE TITLE
[Bug]: The operation was cancelled - Timeout expired

### ISSUE THREAD

**Issue by icodezjb** [MystenLabs/sui#6084]
## Steps to Reproduce Issue

Call move contract on sui testnet

```
sui client call   --gas-budget 10000   --package $package   --module lock   --function add_admin   --args $usdt_cap_lock       0xc05eaaf1369ece51ce0b8ad5cb797b737d4f2eba   --type-args $package::usdt::USDT

[warn] Client/Server api version mismatch, client api version : 0.15.0, server api version : 0.15.1
Failed to execute transaction 2AlQ7xN52MEkT9bTiViwigYRxHzSRq53GtdEkMN/1yg= with error RPC call failed: ErrorObject { code: ServerError(-32000), message: "Failed to execute certificate on a quorum of validators. Validator errors: [\n    \"Unknown error - transport error\",\n    \"Failed to connect with consensus node: Status { code: Unavailable, message: \\\"error trying to connect: tcp connect error: Connection refused (os error 111)\\\", source: Some(tonic::transport::Error(Transport, hyper::Error(Connect, ConnectError(\\\"tcp connect error\\\", Os { code: 111, kind: ConnectionRefused, message: \\\"Connection refused\\\" })))) }\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n]", data: None }

Caused by:
    RPC call failed: ErrorObject { code: ServerError(-32000), message: "Failed to execute certificate on a quorum of validators. Validator errors: [\n    \"Unknown error - transport error\",\n    \"Failed to connect with consensus node: Status { code: Unavailable, message: \\\"error trying to connect: tcp connect error: Connection refused (os error 111)\\\", source: Some(tonic::transport::Error(Transport, hyper::Error(Connect, ConnectError(\\\"tcp connect error\\\", Os { code: 111, kind: ConnectionRefused, message: \\\"Connection refused\\\" })))) }\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n    \"The operation was cancelled - Timeout expired\",\n]", data: None }
```

**Comment by amnn** [comment]
Hi @icodezjb, thanks for the report.  I gather that you are targeting testnet -- the team is investigating some consensus related issues at the moment.  Let me hand this issue over to the current and next oncall for that area, so they can keep you updated.

**Comment by icodezjb** [comment]
resolved in version devnet-0.21.0
